configure.ac

```diff
@@ -1124,7 +1135,7 @@ AC_CHECK_HEADER([math.h],
 if test x"$LIBM" = x ; then
   AC_MSG_WARN([Unable to find working pow function - bgpd may not link])
 fi
-LIBS="$TMPLIBS"
```

Comment:
I assume this is because lua doesn't link without `libm`?

Reply:
fixed